source/config.default.ts

```diff
@@ -0,0 +1,29 @@
+export interface ConfigDefault {
+    statisticBlackList: string[];
+    monitorNames: string[]
+    collectStatisticsInterval: number
+    sendStatisticsInterval: number
+    statsdConfig: StatsdConfig
+}
+
+interface StatsdConfig {
+    prefix: string
+    env: string
+    host: string
+    debug: boolean
+}
+
+export function createDefaultConfig(): ConfigDefault {
```

Comment:
Maybe it makes sense to move the default configuration to json file and interfaces to `config.ts`